samples/job_definitions/data_quality/README.md

```diff
@@ -0,0 +1,57 @@
+# Data Quality Job Definition Sample
+
+This sample demonstrates how to start data quality job definitions using your own data-quality-job-definitions script, packaged in a SageMaker-compatible container, using the Amazon AWS Controllers for Kubernetes (ACK) service controller for Amazon SageMaker.                     
+
+## Prerequisites
+
+This sample assumes that you have already configured an EKS cluster with the ACK operator. It also assumes that you have installed `kubectl` - you can find a link on our [installation page](To do).
+
+You will need an [Endpoint](/samples/endpoint/README.md) configured in SageMaker and also will need to run baselining jobs using the Python SageMaker SDK before you can use it with ACK.
```

Comment:
~~will also need to run baselining jobs using the Python SageMaker SDK before you can use it with ACK~~

you will need to run a baselining job to generate baseline constraints.

it is possible to do it using ACK, later we can add samples for that